spec/games_spec.rb

```diff
@@ -61,4 +61,29 @@
       expect(game_report).to eq(expected_final_report)
     end
   end
+
+  describe '#death_report' do
+    subject(:games){described_class.new('spec/fixtures/example.log')}
```

Comment:
when using "{ }", insert spaces at the start and end.
Ex: `{ described_class.new('file_name.log') }`